app/controllers/api/orgs_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::OrgsController < ApplicationController
+  before_action :authenticate
+  
+  def get
+    render json: { organizations: @user.organizations }, status: :ok
+  end
+
+  private
+  def authenticate
+    authenticate_or_request_with_http_token do |token, options|
```

Comment:
this way is fine, but after testing this locally, `authenticate_or_request_with_http_token` always sets the `Content-Type` of the response object to be HTML when we really want it to be JSON. Replace this with `authenticate_with_http_token` instead